src/theme/DocPage/Layout/Sidebar/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+@use 'src/styles/utility' as *;
+
+:root {
+  --doc-sidebar-width: 283px;
+  --doc-sidebar-hidden-width: 30px;
```

Comment:
Don't we want to use rem function for these aswell?

Reply:
i suggest instead of rem function we can use rem as it is a part of root

Reply:
Good point @sanjam-deriv . In that case we can probably just use 28.3rem and not use the rem function for this.

Reply:
Oh yeah... 😅 I guess keeping it pixel value for now is the best, I think it will get super confusing if we mix the normal rem value with rem function in our code, haha. It's OK for this to keep it pixel.